src/tests/unit/services/VerifiableCredentialService.test.ts

```diff
@@ -276,6 +278,7 @@ describe("VerifiableCredentialService", () => {
 				"nbf": 123456789,
 				"iss": "test-issuer",
 				"iat": 123456789,
+				"jti": expect.anything(),
```

Comment:
What do you think about mocking the `randomUUID` in this function too? Then we can actually expect a value and we can have more confidence in our tests/code

Reply:
Updated.